cli/entryPoint.sh

```
@@ -20,18 +20,18 @@ echo -e ""
 else
     export PS1="\033[32mOPFAB-CLI\033[0m - \033[31m$ENV_NAME\033[0m \w \$ "
 fi
 
-if [ -n "$OPFAB_URL" ]; then
+if [[ -n "$OPFAB_URL" ]]; then
     opfab login $OPFAB_URL $OPFAB_PORT $OPFAB_LOGIN  $OPFAB_PASSWORD
 else
     echo "No configuration provided. Please run 'opfab login' to login in to OperatorFabric."
 fi
 
 cd /opfab/host
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)